app/src/main/java/com/sun/imageeditor/screen/detail/PhotoDetailFragment.kt

```diff
@@ -15,7 +15,7 @@ class PhotoDetailFragment : BaseFragment<FragmentPhotoDetailBinding>(FragmentPho
     }
 
     override fun initView() {
-
+        binding.buttonBack.setOnClickListener { requireActivity().onBackPressed() }
```

Comment:
```suggestion
        binding.buttonBack.setOnClickListener { activity?.onBackPressed() }
```
required activity sẽ bắn exception nếu activity null gây crash